client/src/components/Common/DelayedInput.vue

```diff
@@ -99,6 +102,15 @@ export default {
                 this.$emit("change", this.queryCurrent);
             }
         },
+        selectText() {
+            if (!this.textSelected) {
+                this.$refs.toolInput.select();
+                this.textSelected = true;
+            } else {
+                this.$refs.toolInput.focus();
```

Comment:
Not sure I'm following the re-focus here.  The method is only triggered when `toolInput`'s `focus` is called, correct?

Reply:
If a user clicks in the input a second time, they may want to edit the text that was already there. That's what the refocus is for, since it unselects the text. Otherwise, it was just continually selecting the text that was originally input. Does that make sense?